consensus/helpers.go

```diff
@@ -83,7 +80,7 @@ func (m *consensusModule) getQuorumCertificate(height uint64, step typesCons.Hot
 		pss = append(pss, msg.GetPartialSignature())
 	}
 
-	validators, err := m.getValidatorsAtHeight(height)
+	validators, err := m.GetValidatorsAtHeight(height)
```

Comment:
I don't think `GetValidatorsAtHeight` should be exposed by `Consensus`. Feels like persistence layer logic

Reply:
Agreed, removed, thanks.